### ISSUE TITLE
queued_detector.queued_complete_detector can't found image features

### ISSUE THREAD

**Issue by eduherraiz** [thumbor/thumbor#495]
When I use thumbor detectors, thumbor is able to detect faces and features in the images properly:
> 'thumbor.detectors.face_detector',
> 'thumbor.detectors.feature_detector',

I configured remotecv, and thumbor to get the lazy detection. And it's working, but when I use the: 
> queued_detector.queued_complete_detector

remotecv **detect only faces** in the images.

Configuring only the:
> queued_detector.queued_feature_detector

I can see this error in the remotecv logs:
> ERROR:pyres.worker:(Job{Detect} | remotecv.pyres_tasks.DetectTask | [u'feature', u'www.ushuaiabeachhotel.com/media/ushuaia2015/calltoaction/49_tienda3.jpg', u'www.ushuaiabeachhotel.com/media/ushuaia2015/calltoaction/49_tienda3.jpg']) failed: u'feature'

**Comment by guilhermef** [comment]
Can you debug remotecv's error ?

**Comment by eduherraiz** [comment]
This is the error:
```
Traceback (most recent call last):
  File "/var/thumbor/test/.virtualenvs/test/local/lib/python2.7/site-packages/pyres/worker.py", line 253, in process
    return job.perform()
  File "/var/thumbor/test/.virtualenvs/test/local/lib/python2.7/site-packages/pyres/job.py", line 55, in perform
    return payload_class.perform(*args)
  File "/var/thumbor/test/.virtualenvs/test/local/lib/python2.7/site-packages/remotecv/pyres_tasks.py", line 12, in perform
    points = clz.processor.detect(detection_type, image_data)
  File "/var/thumbor/test/.virtualenvs/test/local/lib/python2.7/site-packages/remotecv/image_processor.py", line 22, in detect
    detector = self.detectors[detector]
KeyError: u'feature'
INFO:pyres.worker:done working
```

**Comment by guilhermef** [comment]
The error is on remotecv's end: https://github.com/thumbor/remotecv/blob/master/remotecv/image_processor.py#L16

I'll correct this now and release a new version.

**Comment by eduherraiz** [comment]
Amazing. I'll happy to test it!

**Comment by guilhermef** [comment]
@eduherraiz I've released remotecv's 1.0.1 version.

**Comment by eduherraiz** [comment]
Thank you! 
I'm think it's not all fixed yet.
Now, when I use the feature detector, works, and detect features:
> queued_detector.queued_feature_detector

But, when I use:
> queued_detector.queued_complete_detector

There is not detecting features (with the same image).

**Comment by eduherraiz** [comment]
Maybe is missing from https://github.com/thumbor/remotecv/blob/master/remotecv/detectors/complete_detector/__init__.py ??

**Comment by guilhermef** [comment]
That's strange, because they're doing the same thing: https://github.com/thumbor/remotecv/blob/master/remotecv/detectors/feature_detector/__init__.py#L23
https://github.com/thumbor/thumbor/blob/master/thumbor/detectors/feature_detector/__init__.py#L37

Can you debug which points it's returning ?
https://github.com/thumbor/thumbor/blob/master/thumbor/detectors/feature_detector/__init__.py#L37

**Comment by eduherraiz** [comment]
I test this works https://github.com/thumbor/remotecv/pull/3

**Comment by guilhermef** [comment]
@eduherraiz I've released the 1.0.2

**Comment by guilhermef** [comment]
Thanks @eduherraiz
